data-extraction/pyDataExtraction/__main__.py

```diff
@@ -0,0 +1,10 @@
+from json import dumps
```

Comment:
Suggest using `import json` to follow "Explicit is better than implicit" principle.